### PR TITLE
Reduce test verbosity.

### PR DESCRIPTION
**Pull request by rabbah**
This could be squashed into #1720.
PG3/156.

### CODE REVIEW

**Comment by rabbah** [comment]
This could be squashed into #1720.
PG3/156.

File: tests/src/whisk/core/controller/test/ControllerTestCommon.scala [hunk 1]
```diff
@@ -26,6 +26,7 @@ import org.scalatest.FlatSpec
 import org.scalatest.Matchers
 
 import akka.event.Logging.{ InfoLevel, LogLevel }
+import akka.event.Logging.ErrorLevel
```
Comment (markusthoemmes) [inline tests/src/whisk/core/controller/test/ControllerTestCommon.scala]: Want to fold the import into the one above \*duck and run\*

Comment (rabbah) [inline tests/src/whisk/core/controller/test/ControllerTestCommon.scala]: https://github.com/openwhisk/openwhisk/pull/1723/commits/cecba0d8d4f13010a572e65ae679808167b76401

**Comment by rabbah** [comment]
@psuter this can be merged with #1720; want me to smash and squash the two or leave to you?

**Comment by psuter** [comment]
Please smash and squash, thanks.

**Comment by rabbah** [comment]
merged with said pr.

**Pull request closed without merging**